ch05.adoc

```diff
@@ -192,6 +192,11 @@ A __grid mapping variable__ provides this information.
 If no __grid mapping variable__ is provided and the coordinate variables for a horizontal grid are not longitude and latitude, then it is required that the latitude and longitude coordinates are supplied via the `coordinates` attribute. 
 Such coordinates may be provided in addition to the provision of a __grid mapping variable__, but that is not required.
 
+When a data variable is representative of cells of non-zero size, and the coordinate variables are not longitude and latitude, __bounds__ variables should be provided for vertices of
+the cell boundaries in the eastings and northings of the grid (see  <<cell-boundaries>>). The __grid mapping variable__ provides then the information to convert bounds
+in latitude and longitude coordinates, and it is optional for the dataset to provide these.
+If no __grid mapping variable__ is provided, then the cell extents in latitude and longitude coordinate system should be provided (see <<bounds-lat-lon>> for one way of doing this).
```

Comment:
I wonder why you say "one way of doing this", not just "(see <<bounds-lat-lon>>)"?

Reply:
The reason is that the section <<bounds-lat-lon>> is for four-sided lat/lon cells, and I thought that the data provider had other options (e.g. 8 vertices per cell).

But I can change this if you prefer, or even remove all mention to <<bounds-lat-lons>> if it makes the convention more robust to future changes.

Reply:
Yes, good point. We could maybe say "(see Section 7.1, Cell Boundaries, especially Example 7.2 for the common case of four-sided cells)"?

Reply:
done.